WasatchNET/AndorEEPROM.cs

```diff
@@ -24,6 +24,9 @@ public class AndorEEPROM : EEPROM
     {
         private AndorSDK andorDriver;
 
+        const short effectiveMinTemp = -60;
```

Comment:
Technically if we coupled one of our units to an external cooler, they should be able to reach -90 or -95 or so.  I think we've just never never had the cooler to test with.

Reply:
good comment, this is just sticky for production's purposes and I think we'd both agree it makes sense to go with the actual rather than theoretical until that actual changes